lib/Dialect/TritonGPU/Transforms/AccelerateMatmul.cpp

```diff
@@ -58,9 +58,8 @@ SmallVector<unsigned> warpsPerTileV2(DotOp dotOp, const ArrayRef<int64_t> shape,
   auto slices = multiRootGetSlice(dotOp, {filter}, {filter});
   bool hasChainedDot = false;
   for (Operation *op : slices) {
-    if (isa<DotOp>(op) && (op != dotOp)) {
-      auto chainedDot = cast<DotOp>(op);
```

Comment:
Are you considering about dot->warpdot (sparsedot)? I'm sure if the logic still applies here then.